.github/workflows/docs.yml

```diff
@@ -46,7 +46,7 @@ jobs:
     runs-on: ${{ needs.determine_runner.outputs.runner_group }}
     steps:
     - uses: actions/checkout@v4
-    - uses: PennyLaneAI/sphinx-action@master
+    - uses: PennyLaneAI/sphinx-action@3.9
```

Comment:
If the `v0.41.1-rc0` branch gets merged back into master, then all of master would get the sphinx build version downgraded as well.

Alternatively, could we include the docstring fixes that is in master in this PR as well?

Reply:
Just an FYI, the branch will not be merged back in, we just use it to create the release. The jax additions will be cherry-picked at the end, but we need the CI to run as it was for the last release.

Reply:
Got it!